<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QWidget" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLineEdit" name="namaLaguLineEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>400</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(63, 63, 63);
color: white;
padding: 6px;
border: none;
border-radius: 6px;
</string>
   </property>
   <property name="placeholderText">
    <string>Nama Lagu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pilihLaguButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>550</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2A2A2A;
color: white;
padding: 6px;
border: none;
border-radius: 6px;
</string>
   </property>
   <property name="text">
    <string>Pilih Lagu</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="playlistTableWidget">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>330</y>
     <width>251</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 255), stop:0.05 rgba(15, 15, 15, 255), stop:0.36 rgba(34, 34, 34, 255), stop:0.6 rgba(53, 53, 53, 255), stop:0.75 rgba(60, 60, 60, 255), stop:0.793427 rgba(67, 67, 67, 255), stop:0.86 rgba(85, 85, 85, 255), stop:0.935 rgba(98, 98, 98, 255));
color: white;</string>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollMode::ScrollPerItem</enum>
   </property>
   <property name="horizontalScrollMode">
    <enum>QAbstractItemView::ScrollMode::ScrollPerItem</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>true</bool>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>34</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Nama Lagu</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nama Artis</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="laguLineEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>550</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(63, 63, 63);
color: white;
padding: 6px;
border: none;
border-radius: 6px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Lagu</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="namaArtisLineEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>450</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(63, 63, 63);
color: white;
padding: 6px;
border: none;
border-radius: 6px;
</string>
   </property>
   <property name="placeholderText">
    <string>Nama Artis</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hapusButton">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>600</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2A2A2A;
color: white;
padding: 6px;
border: none;
border-radius: 6px;
</string>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : transparent;</string>
   </property>
   <property name="text">
    <string>TAMBAH LAGU</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logoutButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>680</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2A2A2A;
color: white;
padding: 6px;
border: none;
border-radius: 6px;
</string>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pilihGambarButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>500</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2A2A2A;
color: white;
padding: 6px;
border: none;
border-radius: 6px;
</string>
   </property>
   <property name="text">
    <string>Pilih Gambar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="coverLineEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>500</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(63, 63, 63);
color: white;
padding: 6px;
border: none;
border-radius: 6px;
</string>
   </property>
   <property name="placeholderText">
    <string>Gambar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tambahButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>600</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2A2A2A;
color: white;
padding: 6px;
border: none;
border-radius: 6px;
</string>
   </property>
   <property name="text">
    <string>Tambah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="keUserButton">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>680</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2A2A2A;
color: white;
padding: 6px;
border: none;
border-radius: 6px;
</string>
   </property>
   <property name="text">
    <string>User</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>251</width>
     <height>721</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/img/Frame 626.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>-10</y>
     <width>1061</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(36, 36, 36);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/img/Frame 604.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>1061</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/img/Frame 604.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>1061</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(36, 36, 36);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/img/Frame 604.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>400</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : transparent;</string>
   </property>
   <property name="text">
    <string>Nama Lagu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>450</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : transparent;</string>
   </property>
   <property name="text">
    <string>Nama Artis</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>1031</width>
     <height>251</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/img/playlist2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_5</zorder>
  <zorder>label_4</zorder>
  <zorder>label_6</zorder>
  <zorder>namaLaguLineEdit</zorder>
  <zorder>pilihLaguButton</zorder>
  <zorder>playlistTableWidget</zorder>
  <zorder>laguLineEdit</zorder>
  <zorder>namaArtisLineEdit</zorder>
  <zorder>hapusButton</zorder>
  <zorder>label_2</zorder>
  <zorder>pilihGambarButton</zorder>
  <zorder>coverLineEdit</zorder>
  <zorder>tambahButton</zorder>
  <zorder>keUserButton</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_3</zorder>
  <zorder>logoutButton</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
